tests/format/Comments/__snapshots__/format.test.js.snap

```diff
@@ -18,7 +18,7 @@ bytes32 private _CACHED_DOMAIN_SEPARATOR;
 /* solhint-enable var-name-mixedcase */
 
 
-    function() {
+    function hello() {
```

Comment:
Q: Is this due to the Slang upgrade?

Reply:
the solidity version needed to be `0.8.29` for the storage layout. but this version doesn't support nameless functions.

Reply:
Thanks for explaining!